docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -907,6 +907,7 @@ _Returns_
 
 -   `?boolean`: Whether the template is valid or not.
 
+
```

Comment:
The pre-commit hooks changed the whitespace in this documentation file; I don't know why.

Reply:
I ran `npm run docs:build` during a rebase and the change was reverted. :man_shrugging: